### PR TITLE
EWC-297 EWC-298 fixed email activation, added user id to signup response

### PR DESCRIPTION
**Pull request by Railag**

### CODE REVIEW

File: src/services/user-service.js [hunk 1]
```diff
@@ -230,8 +230,8 @@ async function _generateAccessToken(transaction) {
 }
 
 function _verifyEmailActivation(emailActivated) {
-  const emailActivation = Config.get("Email:ActivationEnabled") || 'off';
-  if (emailActivation === 'on' && emailActivated === 0)
+  const emailActivation = Config.get("Email:ActivationEnabled");
+  if (emailActivation && emailActivated === 0)
```
Comment (epankou) [inline src/services/user-service.js]: why do you need emailActivated === 0? config setting is of type boolean

Comment (epankou) [reply src/services/user-service.js]: change emailActivated to boolean

Comment (Railag) [reply src/services/user-service.js]: fixed

File: src/services/user-service.js [hunk 2]
```diff
@@ -230,8 +230,8 @@ async function _generateAccessToken(transaction) {
 }
 
 function _verifyEmailActivation(emailActivated) {
-  const emailActivation = Config.get("Email:ActivationEnabled") || 'off';
-  if (emailActivation === 'on' && emailActivated === 0)
+  const emailActivation = Config.get("Email:ActivationEnabled");
+  if (emailActivation && !emailActivated)
```
Comment (epankou) [inline src/services/user-service.js]: please take a look

**Pull request merged**